kotlin-jvm/src/main/kotlin/playground/_main.kt

```diff
@@ -22,6 +22,7 @@ fun main() {
     playground.okio.main()
     playground.retrofit.main()
     playground.sqldelight.main()
+    playground.kotlinx.datetime.main()
```

Comment:
Can you keep this list ordered?
This way we avoid a merge conflict everytime two PR are merged

Reply:
Fixed.